Resources/Locale/ru-RU/_white/reagents/meta/fun.yml

```
@@ -0,0 +1,2 @@
+reagent-name-tranquility = спокойствие
+reagent-desc-tranquility = Излучает странную ауру спокойствия...
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)